iambic/config/wizard.py

```diff
@@ -1934,13 +1907,72 @@ def configuration_github_app_aws_lambda_setup(self):
 
         assert github_app_secrets
 
+        github_app_url = github_app_secrets.get("html_url", "")
+
+        log.info(
+            "We are attempting to open a browser window to the GitHub App installation page.\n"
+            "Please install the app and grant it access to your `iambic-templates` and `iambic-templates-gist` repositories.\n"
+            "If your browser doesn't open, please visit the following URL manually to install the GitHub app and grant access.\n"
+            f"{github_app_url}\n"
+            "Proceed once this is complete.\n\n"
+        )
+        webbrowser.open(github_app_url, new=0, autoraise=True)
+        if not questionary.confirm("Proceed?").unsafe_ask():
+            return
+
         # verify github app control plane access
         github_app_jwt = generate_jwt(github_app_secrets)
         if not verify_access(github_app_jwt):
-            log.error("Unable to continue. Abort.")
+            log.error("We were unable to verify access to Github. Aborting.")
+            return
+
+        github_app_installations = get_github_app_installations(github_app_jwt)
+        if not github_app_installations:
+            log.error("Unable to find Github app installations. Aborting.")
             return
 
-        # safe secret for pem and webhook_url
+        if len(github_app_installations) > 1:
+            log.error(
+                "We found more than one Github app installation. It should only be installed in one Github Organization. Aborting."
+            )
+            return
+
+        github_installation_id = github_app_installations[0]["id"]
+
+        github_app_installation_repos = get_repos_for_installation(
+            github_app_jwt, github_installation_id
+        )
+
+        iambic_templates_repo = next(
+            (
+                r
+                for r in github_app_installation_repos
+                if "iambic-templates" in r["name"] and "-gist" not in r["name"]
+            ),
+            None,
+        )
+        iambic_templates_gist_repo = next(
+            (
+                r
+                for r in github_app_installation_repos
+                if "iambic-templates-gist" in r["name"]
+            ),
+            None,
+        )
+        github_app_secrets.update(
+            {
+                "iambic_templates_repo_url": iambic_templates_repo["clone_url"],
+                "iambic_templates_gist_repo_url": iambic_templates_gist_repo[
+                    "clone_url"
+                ],
+                "iambic_templates_repo_full_name": iambic_templates_repo["full_name"],
+                "iambic_templates_gist_repo_full_name": iambic_templates_gist_repo[
+                    "full_name"
+                ],
+                "github_installation_id": github_installation_id,
```

Comment:
not sure how big the impact is to prefix `github_installation_id` with `iambic_`. The current file format is from GitHub. I think it's best if we prefix all iambic specific stuffs in `iambic_` to avoid future collision.